openapi/crucible-pantry.json

```diff
@@ -5,7 +5,7 @@
     "version": "0.0.0"
   },
   "paths": {
-    "/crucible/pantry/0/job/{id}/is_finished": {
+    "/crucible/pantry/0/job/{id}/is-finished": {
```

Comment:
Same note here, this requires a new generated client. I think that is ok for Omicron, since it will pull in the new Crucible dep (via Propolis) and use the new generated client all at once. But I would like some attention on it.

Reply:
Omicron will need this line changed, I think:
https://github.com/oxidecomputer/omicron/blob/4951c339a8987c757596207c81afdfada8bbf7c7/sled-agent/src/sim/http_entrypoints_pantry.rs#L99